package.json

```diff
@@ -80,6 +80,11 @@
                     "default": 30000,
                     "markdownDescription": "%AWS.configuration.description.samcli.debug.attach.timeout%"
                 },
+                "aws.cdk.explorer.enabled": {
```

Comment:
Food for thought: In CDK activation, the [API](https://code.visualstudio.com/api/references/vscode-api#workspace) `onDidChangeConfiguration` could be used against this configuration value changing in order to generate metrics.

Reply:
I like that suggestion... will start looking into using it